hmda/src/main/scala/hmda/validation/rules/lar/validity/common/V628_1.scala

```diff
@@ -19,10 +19,14 @@ object V628_1 extends EditCheck[LoanApplicationRegister] {
                               OtherHispanicOrLatino,
                               NotHispanicOrLatino,
                               InformationNotProvided,
-                              EthnicityNotApplicable)
+                              EthnicityNotApplicable,
+                              EthnicityNoCoApplicant)
 
   override def apply(lar: LoanApplicationRegister): ValidationResult =
-    when(lar.applicant.ethnicity.otherHispanicOrLatino is empty) {
-      lar.applicant.ethnicity.ethnicity1 is containedIn(validEthnicities)
+    if(lar.applicant.ethnicity.otherHispanicOrLatino.isEmpty) {
+      lar.applicant.ethnicity.ethnicity1 is containedIn(validEthnicities) and (lar.applicant.ethnicity.ethnicity1.code not equalTo(0))
```

Comment:
Simply to match how we write the other edits and because it reads a bit better could you please change this to `lar.applicant.ethnicity.ethnicity1 not equalTo(EmptyEthnicityValue)`